.travis.yml

```diff
@@ -16,7 +16,7 @@ script:
   # Fail the Travis CI build on the first error.
   - set -e
   - mkdir -p build/logs
-  - php vendor/bin/phpcs --standard=PSR12 src
+  - php vendor/bin/phpcs --standard=PSR2 src
```

Comment:
Revert the `PSR-2` coding style and it can avoid `PSR-12` coding style checking to cause the Travis CI build failed.

Reply:
There are many things about PSR-12 that I like, in fact most things. In fact, the only thing I disagree with is that they no longer allow `<?php declare(strict_types=1);`, which I **strongly** feel is the bestest way to declare this.

Here is the corporate workaround:
https://github.com/phpexpertsinc/SimpleDTO/commit/1faa8a43bb0a2ff9f0b53ea762a909e2ad00b8eb

I'm pretty sure all of our projects modded after April 8th 2020 are failing this test.

Reply:
Is it possible to consider using `phpcs.xml` file to ignore `declare(strict_types=1);` checking?

Reply:
@hopeseekr, after searching some solutions about ignoring `declare strict type` checking, here is the proper solution:

The `phpcs.xml` file is as follows:

```XML
<?xml version="1.0"?>
<ruleset name="Custom Standard" namespace="MyProject\CS\Standard">
  <rule ref="PSR12">
    <exclude name="PSR12.Files.FileHeader" />
    <exclude name="PSR12.Files.OpenTag" />
  </rule>
</ruleset>
```